gradle/libs.versions.toml

```diff
@@ -5,13 +5,16 @@ gradle-doctor = "0.8.1"
 google-agp = "8.0.0"
 maven-junit = "5.9.2"
 maven-assertj = "3.24.2"
-maven-ktlint = "0.48.2"
+maven-ktlint = "0.49.0"
 maven-commons = "2.11.0"
 
 [libraries]
 agp-gradle = { module = "com.android.tools.build:gradle", version.ref = "google-agp" }
 kotlin-gradle = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin" }
-ktlint-core = { module = "com.pinterest.ktlint:ktlint-core", version.ref = "maven-ktlint" }
+ktlint = { module = "com.pinterest:ktlint", version.ref = "maven-ktlint" }
+ktlint-rule-engine = { module = "com.pinterest.ktlint:ktlint-rule-engine", version.ref = "maven-ktlint" }
+ktlint-cli-ruleset-core = { module = "com.pinterest.ktlint:ktlint-cli-ruleset-core", version.ref = "maven-ktlint" }
+ktlint-cli-reporter = { module = "com.pinterest.ktlint:ktlint-cli-reporter", version.ref = "maven-ktlint" }
```

Comment:
Needed for `ReporterV2`/`ReporterProviderV2`, `KtlintCliError` (used in reporters)